lib/logstash/config/mixin.rb

```diff
@@ -113,6 +114,10 @@ def config_init(params)
       instance_variable_set("@#{key}", value)
     end
 
+    @logger.terminal(I18n.t("logstash.plugin.activated_sucessfully", 
```

Comment:
I'm wondering if this is totally necessary for all plugins.

I'm in favor of having stdin for sure do this log to say "Standard input is ready. Please type some things at your keyboard and press enter when you want to submit" or something

Do we need all plugins to report "ready" ?